### ISSUE TITLE
The facilitation of discussions about the 3Dness of Digital Twins.

### ISSUE THREAD

**Issue by ogcscotts** [ogcscotts/TC-Meeting-topics#240]

**Comment by chris-little** [comment]
@ogcscotts Shouldn't this be the 4Dness? I cannot see how a Digital Twin would be useful unless it 'predicts' something.

**Comment by ogcscotts** [comment]
@chris-little of course, I'm going to expand to "nDness" because it's all of that other information that makes up the internal organs of the twin.

**Comment by PeterParslow** [comment]
From engagement with the JTC1 activities on smart cities & digital twins, there is a sort of maturity curve. For some requirements, the city/community/park whatever doesn't need 3D data. Whether in this case what they do need qualifies as a digital twin is another matter! For many, it's the sensor/actuator feedback loop that makes the thing a "digital twin", not the "shape" of the geographic representation.

Regarding "predicting", that's certainly one use case. Others include monitoring, influencing the environment, post-issue root cause analysis, future looking 'what if' analysis - these latter two, with their time dimensions, imply the ability to clone as many "twins" as needed & then modify aspects of them, independently of the "main twin" being in live use to push modifications back into the real world - change settings on an engine, or traffic lights in a street. All that cleverness / usefulness is independent of 3D.

Here are a couple of potentially useful JTC 1 quotes:

"A Digital Twin is a digital model of a particular physical element or a process with data connections that enable convergence between the physical and virtual states _at an appropriate rate of synchronisation_" (my emphasis, because it's about timeliness) ISO 23247, CD quoted by JTC1/WG11

& from TC 211:

"A digital twin of a city is a tool that a smart city can use to help it achieve its objectives.
Those objectives are often aspects of becoming more sustainable: financially, socially, or environmentally."

It should be able to help with operational, critical, analytical, and strategic "levels of insight". But how much information it needs to achieve that will depend on the particular goals (ISO 30146:2019)
